### PR TITLE
Prevent replication flat loops

### PR DESCRIPTION
**Pull request by Atlante45**
This PR prevents packets from getting trapped in a replication loop.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1006/HighFidelity-Beta-PR10786-92efce24a32f61040a3b9358961350de88a21c11.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1006/HighFidelity-Beta-PR10786-92efce24a32f61040a3b9358961350de88a21c11.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1022/HighFidelity-Beta-PR10786-e14130b09a5343f4562c94dd7cce852b3c31597c.exe

File: assignment-client/src/audio/AudioMixer.h [hunk 1]
```diff
@@ -51,6 +51,11 @@ class AudioMixer : public ThreadedAssignment {
     static const QVector<ReverbSettings>& getReverbSettings() { return _zoneReverbSettings; }
     static const std::pair<QString, CodecPluginPointer> negotiateCodec(std::vector<QString> codecs);
 
+    static bool shouldReplicateTo(const Node& from, const Node& to) {
+        return to.getType() == NodeType::DownstreamAvatarMixer &&
```
Comment (birarda) [inline assignment-client/src/audio/AudioMixer.h]: DownstreamAvatar?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1038/HighFidelity-Beta-PR10786-71d2540a239a0b3467075a5eb9096baa0631b43a.dmg

**Comment by Atlante45** [comment]
test this please

**Comment by Atlante45** [comment]
Given, the difficulty of the setup to QA this, @birarda and I just QA tested it ourselves.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1043/HighFidelity-Beta-PR10786-71d2540a239a0b3467075a5eb9096baa0631b43a.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1043/HighFidelity-Beta-PR10786-71d2540a239a0b3467075a5eb9096baa0631b43a.dmg

**Pull request merged**